configs/example/gpufs/system/amdgpu.py

```diff
@@ -185,3 +185,11 @@ def connectGPU(system, args):
         system.pc.south_bridge.gpu.DeviceID = 0x6863
     else:
         panic("Unknown GPU device: {}".format(args.gpu_device))
+
+    system.pc.south_bridge.gpu.Status = 0x0290
```

Comment:
It would be nice if you could add a comment explaining these magic values here

Reply:
Added comments